Content.Client/Pointing/PointingSystem.cs

```diff
@@ -49,6 +46,9 @@ private void AddPointingVerb(GetVerbsEvent<Verb> args)
         if (_mobState.IsIncapacitated(args.User))
             return;
 
+        if (HasComp<SleepingComponent>(args.User))
+            return;
+
```

Comment:
This should be raising an event / checking CanInteract rather than hardcoding in a bunch of systems.